tests/ui/missing-function/replaced-description/expected

```diff
@@ -1,3 +1,3 @@
 Status: FAILURE\
-Description: "Function `<alloc::string::String as core::clone::Clone>::clone` with missing definition is unreachable"
```

Comment:
Why did this change?

Reply:
I believe because the Kani library now uses a vector (for storing the task list) and it just so happens to be checked before the String's clone method.